.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
```

Comment:
```suggestion
      - "*.*.*"
```

Reply:
What do you think about something like `[0-9]+.[0-9]+.[0-9]+`?

Reply:
Yeah, that's better